libs/webb-ui-components/src/utils/getRoundedAmountString.ts

```diff
@@ -26,7 +26,8 @@ export function getDecimals(digit: number): number {
 export function getRoundedAmountString(
   num: number | undefined,
   digits = 3,
-  roundingFunction: numbro.Format['roundingFunction'] = Math.floor
+  roundingFunction: numbro.Format['roundingFunction'] = Math.floor,
+  removeLengthConstraint: boolean | undefined = undefined
```

Comment:
You should use a more generic parameter, `formatOpts: numbro.Format`, and spread that option within the format call as `...formatOpts`. Be aware that this could potentially break the `roundingFunction` option. To handle this, locate the area where we override it and perform the necessary updates.

```ts
export function getRoundedAmountString(
  num: number | undefined,
  digits = 3,
  formatOpts: numbro.Format = { roundingFunction: Math.floor }
): string {
  if (num === 0) {
    return '0';
  }

  if (!num) {
    return '-';
  }

  if (num < 0) {
    return '< 0';
  }

  const decimals = getDecimals(digits);
  if (num < decimals) {
    return `< ${decimals}`;
  }

  return numbro(num).format({
    average: num > 1000,
    totalLength: num > 1000 ? 3 : 0,
    mantissa: digits,
    optionalMantissa: true,
    trimMantissa: true,
    thousandSeparated: true,
    ...formatOpts,
  });
}

```